core/unit_test/TestGraph.hpp

```diff
@@ -792,25 +809,40 @@ struct GraphNodeTypes {
 
   // Type of an aggregate node.
   using aggregate_t = typename Kokkos::Impl::GraphImpl<Exec>::aggregate_impl_t;
+
+  // Type of a then node.
+  using then_t =
+      Kokkos::Impl::GraphNodeThenImpl<Exec, ThenFunctor<Kokkos::View<int>>>;
 };
 
 constexpr bool test_is_graph_kernel() {
-  using types = GraphNodeTypes<TEST_EXECSPACE>;
+  using types = GraphHelpers<TEST_EXECSPACE>;
```

Comment:
hmm I liked `GraphNodeTypes` better

Reply:
OK, I'll revert to `GraphNodeTypes` :wink:

Reply:
Done :heavy_check_mark: